test/integration/connect/envoy/case-mesh-to-lambda/config_entries.hcl

```diff
@@ -0,0 +1,34 @@
+config_entries {
+  bootstrap {
+    kind = "terminating-gateway"
+    name = "terminating-gateway"
+
+    services = [
+      {
+        name = "l2"
+      }
+    ]
+  }
+  bootstrap {
+    kind = "service-defaults",
+    name = "l1",
+    protocol = "http"
+    meta {
+      "serverless.consul.hashicorp.com/v1alpha1/lambda/enabled" = "true"
+      "serverless.consul.hashicorp.com/v1alpha1/lambda/arn" = "arn:aws:lambda:us-west-2:977604411308:function:consul-envoy-integration-test"
```

Comment:
We have some options on how we deploy a Lambda function for the integration tests to interact with:

1. Create a long-lived, well-known Lambda function outside of CI. Then reference this function in the tests. This approach is simple and will work as long as no one deletes the Lambda function.
1. Dynamically create the function when the tests run as part of the setup. This could be done via the `aws` CLI or Terraform. This is potentially more flexible and would keep the Lambda function code colocated with the test; however, it introduces more dependencies and complexity within the test setup.

For now I've implemented option 1. I plan to stick with this approach unless there are objections.

Reply:
IMO 1 is better because it will be significantly faster and less flaky.

Reply:
I think it's worth adding a comment about this to the code

Reply:
Added to the CircleCI config file.